CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+# v1.0.8 (2021-04-12)
+
+## Deprecation Notice
+
+Bottlerocket 1.0.8 is the last release where we plan to support the Kubernetes 1.15 variant, `aws-k8s-1.15`.
```

Comment:
I'd prefer to say:
```suggestion
Bottlerocket 1.0.8 is the last planned release that will support the Kubernetes 1.15 variant, `aws-k8s-1.15`.
```

Since we may do unplanned releases with critical security or bug fixes prior to the EOL in early May.

Reply:
Discussed out-of-band - it seems like we are trying to express the same thing, so I'm fine with this wording and we can adjust it if it causes confusion.